esi_leap/manager/service.py

```diff
@@ -130,6 +140,62 @@ def _expire_offers(self):
                     offer.status = statuses.ERROR
                     offer.save()
 
+    def _notify_lease_expire(self):
+        LOG.info('Checking for leases expire soon')
+        leases = lease_obj.Lease.get_all(
+            {'status': [statuses.ACTIVE, statuses.CREATED,
+                        statuses.WAIT_EXPIRE, statuses.WAIT_FULFILL]},
+            self._context)
+        now = timeutils.utcnow()
+        for lease in leases:
+            if lease.end_time - now <= datetime.timedelta(1):
```

Comment:
If this runs every 6 hours, won't a user get up to four emails notifying them that their lease is about to expire?

Reply:
I changed it to 24 hours, so users will get at most one email for one day